torchao/prototype/awq/example.py

```diff
@@ -6,17 +6,23 @@
 import argparse
 import time
 
+import lm_eval
 import torch
 from datasets import load_dataset
+from lm_eval import evaluator
+from lm_eval.models.huggingface import HFLM
+from torch._inductor import config as inductor_config
 from tqdm import tqdm
 from transformers import AutoModelForCausalLM, AutoTokenizer, TorchAoConfig
 
 from torchao.prototype.awq import (
     AWQConfig,
 )
-from torchao.quantization import (
-    quantize_,
-)
+from torchao.quantization import Int4WeightOnlyConfig, quantize_
+
+inductor_config.cpp_wrapper = True
+inductor_config.max_autotune = True
+inductor_config.max_autotune_gemm_backends = "CPP,ATEN"
```

Comment:
This script will also be used for CUDA. So, I think `triton` is needed here. Or let's simply remove this line to use the default ones.